src/webapp/src/auth/AuthenticationOptions.tsx

```diff
@@ -1,7 +1,68 @@
+import { Button } from '@small-ads/ui';
 import React from 'react';
+import styled from 'styled-components';
+import { FacebookSquare } from '@styled-icons/boxicons-logos';
+
+const LoginScreen = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: calc(100vh - 64px);
```

Comment:
The menu will always be 64px. Could we extract this to a constant?

Reply:
If Navbar isnt living in the ui then we cant be importing its height from the ui. We can have a `constants.ts` in the Navbar component directory

Reply:
Right, I thought we could export also constants from the `ui`. But I can see how this is better. thanks